File: android/data/src/main/java/pl/rspective/data/entity/Question.java
package pl.rspective.data.entity;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Question implements Serializable {

    @SerializedName("Id")
    private long id;

    @SerializedName("Text")
    private String text;

    private long userAnswerId;

    @SerializedName("Answers")
    private List<Answer> answers;

    public long getUserAnswerId() {
        return userAnswerId;
    }

    public void setUserAnswerId(long userAnswerId) {
        this.userAnswerId = userAnswerId;
    }

    public long getId() {
        return id;
    }

    public String getText() {
        return text;
    }

    public List<Answer> getAnswers() {
        return answers;
    }
}

File: front-end/Rspective.FurryOctoNinja.Web/Models/Question.cs
using System.Collections.Generic;

namespace Rspective.FurryOctoNinja.Web.Models
{
    public class Question
    {
        public int Id { get; set; }

        public string Text { get; set; }

        public IEnumerable<Answer> Answers { get; set; }
    }
}